client/src/components/custom-dropdown.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Container, Dropdown } from 'react-bootstrap';
+import DropdownToggle from 'react-bootstrap/DropdownToggle';
+import MyDatePicker from './date-picker';
+
+class DropdownWithDate extends React.Component {
+  render() {
+    /* eslint-disable */
```

Comment:
You should avoid to disable all eslint rules in page because it allows you doing a lot of mistakes
If you want to disable some rule you should use `// eslint-disable-next-line some-rule` 

Also you can use editor with autocorrection (e.g. vscode with eslint extension and setting ```"editor.codeActionsOnSave": {
          "source.fixAll": true
    }```)
or use script `npm run lint:fix`

But eslint --fix doesn't fix all mistakes, so you should rewrite your code or update .eslintrc file